### PR TITLE
Log long operations on iptables

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref #48107

### CODE REVIEW

**Comment by shyamjvs** [comment]
/lgtm
Thanks.

**Comment by shyamjvs** [comment]
You'll probably need to add all the milestone-related labels if you want this change in soon.

**Comment by wojtek-t** [comment]
Fixed compile errors - reapplying label.

**Comment by wojtek-t** [comment]
Fixed gofmt - reapplying label.

**Review by dcbw — APPROVED** [review]

File: pkg/util/iptables/iptables.go [hunk 1]
```diff
@@ -317,6 +318,9 @@ func (runner *runner) SaveInto(table Table, buffer *bytes.Buffer) error {
 	runner.mu.Lock()
 	defer runner.mu.Unlock()
 
+	trace := utiltrace.New("iptables save")
+	defer trace.LogIfLong(2 * time.Second)
```
Comment (krmayankk) [inline pkg/util/iptables/iptables.go]: @wojtek-t curious how does this work ? is this running the iptables save command or that is just naming the trace ?

Comment (wojtek-t) [reply pkg/util/iptables/iptables.go]: It just adds a trace and ensures that if whole operation took more than 2s, it will be logged.

**Comment by wojtek-t** [comment]
I just added an additional "Step" instruction to make it clear when we acquired a lock (and if the operation was long due to waiting for lock or due to being long itself).
Since it's a trivial change, I'm self-applying lgtm label.

**Pull request merged**